gap/digraph.gi

```diff
@@ -554,7 +554,8 @@ function(D)
     Append(str, "multi");
   fi;
 
-  if HasIsTournament(D) and IsTournament(D) and n > 1 then
+  if not (HasIsCycleDigraph(D) and IsCycleDigraph(D))
```

Comment:
What's the overlap between being a tournament and being a cycle, some corner case?

Reply:
Yes, a corner case. The only overlap is that a cycle digraph with three vertices is a tournament (although not all tournaments with three vertices are cycle digraphs).

This bit of code stops `<(im)mutable cycle tournament with 3 vertices>` from showing, in favour of the simpler `<(im)mutable cycle digraph with 3 vertices>`, in this case.